src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -36,57 +37,74 @@ <h1>{{ title }}</h1>
 {% elif state == "confirmation_invalid" %}
   {% trans email=request.user.email %}
   The provided code is invalid, unauthorized, expired or incomplete. Please use the link in the email sent to your email address {{ email }}. If the code is still not working, please request a new email.
-
-
   {% endtrans %}
 {% endif %}
 </div>
-{% if found_emails|length > 0 %}
+
+
+{% if render_table %}
 <div class="verify-email-table">
   <p>
     {% trans %}
-      We have attempted to send your verification email.
-      Below are the confirmation records we found and their associated delivery statuses.
+      <strong>Email history:</strong> The table below shows all emails we have attempted to send to you in the last 2 days.
     {% endtrans %}
   </p>
   <table border=1 frame=void rules=rows>
-    <thead>
-      <tr>
-        <th>{{ _('Date') }}</th>
-        <th>{{ _('From') }}</th>
-        <th>{{ _('To') }}</th>
-        <th>{{ _('Subject') }}</th>
-        <th>{{ _('Status') }}</th>
-      </tr>
-    </thead>
-    <tbody>
-    {% for email in found_emails %}
-      <tr>
-        <td>
-          {{ email.statusDate }}
-        </td>
-        <td>
-          {{ email.from }}
-        </td>
-        <td>
-          {{ email.to }}
-        </td>
-        <td>
-          {{ email.subject }}
-        </td>
-        <td>
-          {{ email.status }}
-        </td>
-      </tr>
-    {% endfor %}
-    </tbody>
+    {% if found_emails|length > 0 %}
+      <thead>
+        <tr>
+          <th>{{ _('Date') }}</th>
+          <th>{{ _('From') }}</th>
+          <th>{{ _('To') }}</th>
+          <th>{{ _('Subject') }}</th>
+          <th>{{ _('Status') }}</th>
+        </tr>
+      </thead>
+      <tbody>
+      {% for email in found_emails %}
+        <tr>
+          <td>
+            {{ email.statusDate }}
+          </td>
+          <td>
+            {{ email.from }}
+          </td>
+          <td>
+            {{ email.to }}
+          </td>
+          <td>
+            {{ email.subject }}
```

Comment:
Should we truncate that for potential display issues with long subjects ?

Reply:
I was going to say it doesn't matter since we control the subject, but now we are displaying all emails so yeah maybe it's a good point. I can add a css truncation.

Reply:
Done.